src/sbt-test/dumpLicenseReport/custom-report/test

```diff
@@ -1,4 +1,5 @@
 > dumpLicenseReport
 $ exists target/license-reports/lreport.md
 > check
-$ exists target/license-reports/test-config.html
\ No newline at end of file
+$ exists target/license-reports/test-config.html
```

Comment:
Yes, it's confusing.  The only change in that line is I added a newline at the end.